facebook/src/main/java/com/facebook/GraphRequest.java

```diff
@@ -1706,7 +1706,8 @@ final static void serializeToUrlConnection(
 
         OutputStream outputStream = null;
         try {
-            outputStream = new BufferedOutputStream(connection.getOutputStream());
+            outputStream = connection.getOutputStream();
```

Comment:
Cool, can you submit the pr with just the InputStream changes, and I can merge? Thanks.